cmd/argocd/commands/app.go

```diff
@@ -1819,8 +1819,10 @@ func waitOnApplicationStatus(acdClient apiclient.Client, appName string, timeout
 	defer argoio.Close(conn)
 	app, err := appClient.Get(ctx, &applicationpkg.ApplicationQuery{Name: &appName})
 	errors.CheckError(err)
+	fmt.Fprintf(os.Stdout, "started watch, resourceVersion: %s\n", app.ResourceVersion)
 	appEventCh := acdClient.WatchApplicationWithRetry(ctx, appName, app.ResourceVersion)
 	for appEvent := range appEventCh {
+		fmt.Fprintf(os.Stdout, "received event: %v\n", appEvent)
```

Comment:
This is just for debugging right?

Reply:
ops, second commit is just for debugging - removing it.